portal/.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     'object-curly-newline': ['error', {
       consistent: true,
     }],
+    'no-restricted-syntax': 0,
```

Comment:
As in the other config file, please can we change `0` to `'off'`? And on line 18 and 19?

Reply:
Have updated.